src/components/page.tsx

```diff
@@ -46,19 +46,27 @@ const headerText = css`
   flex-shrink: 1;
   text-overflow: ellipsis;
   overflow: hidden;
+
+  & a {
+    color: white;
+
+    &:hover {
+      text-decoration-color: ${greenOnPurple};
+    }
+  }
 `
 
 type Props = Merge<
   JSX.HTMLAttributes,
   {
     name: ComponentChildren
-    back: string | (() => void) | false
+    showBackButton?: boolean
     class?: string
     wrapperClass?: string
   }
 >
 
-const Header = ({ back, name }: Omit<Props, 'class'>) => {
+const Header = ({ name, showBackButton }: Omit<Props, 'class'>) => {
```

Comment:
```suggestion
const Header = ({ name, showBackButton = true }: Omit<Props, 'class'>) => {
```

@varCepheid would it be more expressive to set the default value to true? Then people who read the function signature can see that the parameter defaults to true, and the code below that uses it could be simplified to be easier to read too.